src/modality-specific-files/motion.md

```diff
@@ -0,0 +1,181 @@
+For example, if the position time series of multiple optical markers is processed via one recording unit, this can be defined as a single tracking system.
+Note that it is not uncommon to have multiple tracking systems to record at the same time.
+
+Each tracking system MUST have its own `*_tracksys-<label>_motion.tsv` file, where `<label>` is a user definded key word to be used to identify each file belonging to a tracking system.
+This is especially helpful when more than one tracking system is used.
+One column in the `*_tracksys-<label>_motion.tsv` file represents one data channel.
+The ordering of columns MUST match the order of rows in the `*_channels.tsv` file for unambiguous assignment.
+All relevant metadata about a tracking systems is stored in accompanying sidecar `*_tracksys-<label>_motion.json` file.
+
+The source data from each tracking system in their original format, if different from `.tsv`,
+can be stored in the [`/sourcedata` directory](../common-principles.md#source-vs-raw-vs-derived-data).
+The original data format MAY hold more metadata than currently specified in the `*_motion.json` file.
+
+When multiple tracking systems are used to record motion or motion capture is used alongside the recording of other BIDS modalities, it may be necessary to temporally synchronize the recordings.
+To save the differences between recording onsets, the `acq_time` column of the [`scans.tsv`](../modality-agnostic-files.md#scans-file) files can be used.
+
+To store events alongside motion data when there are multiple tracking systems simulatenously in use, it is RECOMMENDED to designate a tracking system to the events file.
+Such an events file name SHOULD include the `tracksys` key and looks like `sub-<label>[_ses-<label>]_task-<label>[_acq-<label>]_tracksys-<label>[_run-<index>]_events.tsv`.
+Event latencies can then be related to motion samples of multiple tracking systems also by using `acq_time` column entries in the `scans.tsv`.
+The same principle applies when the events file is saved alongside a simultaneously recorded non-motion data (for example EEG).
+
+### Sidecar JSON (`*_motion.json`)
+
+#### Task information
+
+{{ MACROS___make_sidecar_table("motion.motionTaskInformation") }}
+
+#### Hardware information
+
+{{ MACROS___make_sidecar_table("motion.motionHardware") }}
+
+#### Insitution information
+
+{{ MACROS___make_sidecar_table("motion.motionInstitutionInformation") }}
+
+#### Motion specific fields
+
+Motion specific fields MUST be present:
+
+{{ MACROS___make_sidecar_table("motion.motionRequired") }}
+
+Motion specific fields SHOULD be present:
+
+{{ MACROS___make_sidecar_table("motion.motionRecommended") }}
+
+#### Example `*_tracksys-<label>_motion.json`
+
+```JSON
+{
+ "SamplingFrequency": 60,
+ "SamplingFrequencyEffective": 60.00197437,
+ "TaskName": "BIDS Motion fictive example",
+ "TrackingSystemName": "imu1",
```

Comment:
might need adjustment to show that (now, after our discussion) this field MAY be used to record a more human readable / complete description corresponding to the `tracksys` entity